.mailmap

```diff
@@ -1649,3 +1649,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
 袁野 (Yuan Ye) <yuanyelele@tutanota.com>
+HeeJae Chang <hechang@microsoft.com>
```

Comment:
You need to rerun `bin/mailmap_check.py` so that the file is sorted alphabetically.

Reply:
done.